src/System.Private.CoreLib/shared/System/Numerics/Matrix4x4.cs

```diff
@@ -1761,7 +1759,6 @@ public static Matrix4x4 Transform(Matrix4x4 value, Quaternion rotation)
         /// <returns>The transposed matrix.</returns>
         public static unsafe Matrix4x4 Transpose(Matrix4x4 matrix)
         {
-#if HAS_INTRINSICS
```

Comment:
We only build for platforms where the intrinsics are exposed, so the `#ifdefs` aren't needed.
* Noting that they are not necessarily supported everywhere and some platforms, such as mono, will treat `IsSupported` as `false` (as with other places in shared corelib code).